src/requirementslib/models/setup_info.py

```diff
@@ -114,6 +114,16 @@ def pep517_subprocess_runner(cmd, cwd=None, extra_environ=None):
     )
 
 
+def read_source(path, encoding="utf-8"):
```

Comment:
Would you mind moving this to `models/utils.py` and checking that we are doing this correctly everywhere?

Reply:
Moved to `utils`

I searched all calls to `open` and all are OK.